Common++/header/MacAddress.h

```diff
@@ -107,16 +106,34 @@ namespace pcpp
 			if(m_IsValid)
 			{
 				#if _MSC_VER >= 1800
-				std::copy(octets.begin(), octets.end(), stdext::checked_array_iterator<uint8_t*>(m_Address, 6));
+				std::copy(octets.begin(), octets.end(), stdext::checked_array_iterator<uint8_t*>(m_Address, sizeof m_Address));
 				#else
 				std::copy(octets.begin(), octets.end(), std::begin(m_Address));
 				#endif
 			}
 
 			return *this;
 		}
+
+		/**
+		 * Returns the size of raw data
+		 * @return The size of raw data
+		 */
+		static constexpr size_t rawDataSize() { return 6; }
+#else
+		/**
+		 * Returns the size of raw data
+		 * @return The size of raw data
+		 */
+		static size_t rawDataSize() { return 6; }
```

Comment:
are you using these new methods anywhere in the code?

Reply:
Yes, I will use this methods. In my project the MAC addresses are saved into the binary file queue and later are loaded into database. My current code:

```
uint8_t mac[6];
macData->value.mac.copyTo(mac);
mFieldMac.assign(mac, sizeof mac);

```
1. It is not a good idea to use directly defined size.
2. I am forced to copy MAC's bytes into temporary array to pass them into the function.

The class has `void copyTo(uint8_t* arr)` method therefore it is good to know the size of the buffer.

Other way is to create a constant instead of a method: enum { RawDataSize = 6 }

Reply:
Other way is to create a constant instead of a method: enum { RawDataSize = 6 }

Reply:
As I replied earlier, we should consider converting this from a method to something else, preferably a static const member or a macro

Reply:
I have removed rawDataSize method and let's not create a new constant.